schema/signature.go

```diff
@@ -0,0 +1,24 @@
+package schema
+
+import (
+	"github.com/zclconf/go-cty/cty"
+	"github.com/zclconf/go-cty/cty/function"
+)
+
+type FuncSignature struct {
```

Comment:
```suggestion
type FunctionSignature struct {
```

This would IMO better align with `function.Function` in `go-cty`: https://pkg.go.dev/github.com/zclconf/go-cty@v1.12.1/cty/function#Function